### ISSUE TITLE
Stoploss and minimal_roi

### ISSUE THREAD

**Issue by shotuhhh** [freqtrade/freqtrade#9539]
## Describe your environment

  * Operating system: ____Linux
  * Python Version: _____ 3.9(`python -V`)
  * CCXT version: _____ 4.1(`pip freeze | grep ccxt`)
  * Freqtrade Version: ____Last version  (`freqtrade -V` or `docker compose run --rm freqtrade -V` for Freqtrade running in docker)
  
## Your question
Hello, thanks for last answer to my question.
I would like to know how I can set my Stoploss and minimal_roi percentage(%) in minimum and maximum range or maybe another variant that I can set my stop loss and minimal_roi from 1% to 2.% and my Stoploss and minimal_roi work from range 1% to 2% (1.1%, 1.2%, 1.3%,.....2%) ?

This is my stop loss and minimal_roi but here Stoploss and minimal_roi but here both work in different way % than I set before not work.
Please help with this!

Best Regards

minimal_roi ={
   "0": 0.011,
   "": 0.015
},
    stoploss = -0.025

**Comment by hippocritical** [comment]
you seem to have a typo in minimal_roi
https://www.freqtrade.io/en/stable/strategy-customization/#minimal-roi

stoploss seems to be proper
https://www.freqtrade.io/en/stable/strategy-customization/#stoploss

keep in mind that if you use leverage, then this will move the stoploss + roi.

Stoploss and roi have to be seen including your leverage

**Comment by shotuhhh** [comment]
Thanks for answering.
But I can't find information about what I want .
I want that Stoploss and minimal_ro percentage at least work from 1%...to 2%, maximum for example of Stoploss and minimal_roi will ne 2%

Please help I read Ursls that you send but no work.

**Comment by hippocritical** [comment]
please show me why you think it doesnt work
and +-2% (stoploss / roi) will never work in live, anything below 5% is already really hard

**Comment by shotuhhh** [comment]
I can send my strategy file if you but if I set my stop loss or minimal_roi for exam 2% but it's work then 2% for both.

Please help how I can set for exam a range among 1% to 2% for both (stop loss or minimal_roi) or maybe set maximum percentage 2% that working stop loss and minimal_roi no more than 2% ?

Best Regards

**Comment by shotuhhh** [comment]
Please help with this

**Comment by xmatthias** [comment]
To be quite honest, even after 5 messages, i still have no idea what you're actually asking.

the following settings will "bracket" your trade to +-2% - limiting both the loss and the profit to roughly 2%.

```
minimal_roi ={
"0": 0.02
},
stoploss = -0.02
```

Now in a live (and depending on settings, most likely also dry) environment, you'll not get "exactly" 2% in most cases. 
This is because of the way orders fill. While we could place an order at -2%  exactly - if the price is further down (prices can skip levels if it's a low volume market), your order would most likely not fill - or would fill a lot later (if at all) - potentially leading to huge losses.

As such, the actual exit price is determined at the time of order placement, and may be at 2% - or at 2.1% ... depending on where the actual orderbook price is at that moment in time.


If this information is not what you're after (it's a guess, as the question is absolutely not clear to me), then please
* make sure to describe what you're asking in a clear way (if you're not a native english speaker, maybe write them in your native language and use a tool like deepl or google translate to translate it to english).
* if applicable, provide logs to show what is happening, while describing what you think should happen

**Comment by shotuhhh** [comment]
I am sorry for my English.
In short I want that working my stop loss and minimal_roi work no more my set percentage(2%...) 

How I can set my stoploss or minimal_roi in minimum and maximum % of working for example min 1% max 2% and working my stop loss or minimal roi in this range.

**Comment by stash86** [comment]
Are we talking about hyperopt here?

**Comment by shotuhhh** [comment]
Nooo nooo I say how I can set my stop loss or minimal_roi by the end not working more than percentage that set.
For example if I set my stop loss 2%, but it's work more 2%.
My question is.

How I can set my stop loss that work NO more 2% ??
Only this

**Comment by stash86** [comment]
Slippage is out of our reach. When your stoploss is bigger than 2%, that's because the market is moving faster than the bot can put order. You can try stoploss on exchange if you want (almost) exact stoploss percentage

**Comment by stash86** [comment]
Something for you to read.

https://www.investopedia.com/terms/s/slippage.asp

**Comment by shotuhhh** [comment]
Ok thanks.
Maybe I use a very small % stop loss for getting maximum slippage 2% or ?

**Comment by stash86** [comment]
Use stoploss_on_exchange feature. Read more on the docs. You can't really control how much slippage gonna happen.

**Comment by stash86** [comment]
And just personal experience, 2% stoploss is too tight in my opinion if your strategy is trying to catch pairs that are going down, expecting it to go up. And if you do backtest, I believe it should show you that you have many stoploss triggered

**Comment by shotuhhh** [comment]
Thank you very much for such best advice and answering the questions.
Maybe you teach me personally of course no for free?

**Comment by xmatthias** [comment]
I'll assume this question as answered now.

Please don't use issues in our organization as job-board, thanks.

**Comment by shotuhhh** [comment]
I am sorry but I am felt awkward for giving many questions that's I say about answering for money.
Thanks
Best Rewards

**Comment by stash86** [comment]
> Thank you very much for such best advice and answering the questions. Maybe you teach me personally of course no for free?

You can read this https://botacademy.ddns.net/

If there is a topic not found there, let me know and I will try to write about it. But sorry, they are all in english. So you will need to work on your english as well.
